doc/guides/tips-tricks-and-troubleshooting.md

```diff
@@ -0,0 +1,74 @@
+# Tips, Tricks & Troubleshooting
```

Comment:
(Not being cheeky but) do you have a suggestion? (Please don't say FAQ...)
Maybe "Tips & Best Practices"

P.S. usually the main problem with the "junk drawer" is finding stuff, but here we have the search function, for example if you search `unknown encoding: cp65001` this will pop up.